README.md

```diff
@@ -4,10 +4,21 @@ This repository contains several sample applications that demonstrate various Ok
 
 Each sample makes use of the [Okta React Library][].
 
-Before running the samples, you will need to gather the following information from the Okta Developer Console:
+If you haven't done so already, register for a free account at [developer.okta.com](https://developer.okta.com/). Select **Create Free Account** and fill in the forms to complete the registration process. Once you are done and logged in, you will see your Okta Developer Console. 
 
-- **Client Id** - The client ID of the SPA application that you created earlier. This can be found on the "General" tab of an application, or the list of applications.  This identifies the application that tokens will be minted for.
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+> **Tip**: You can also create an account using the [Okta CLI](https://github.com/oktadeveloper/okta-cli) and `okta register`. To create an app, run `okta apps create` and use the settings below.
+
+Register your application by selecting **Applications** > **Add Application**. On the next screen, choose **Single Page App** and click **Next**.
+
+On the following screen, edit the application settings. For React applications running in developer mode, the port number should be 3000. Configure your app as follows:
+
+* **Base URI**: `http://localhost:3000`
```

Comment:
All the URLs configured in these samples seem to be using port 8080. Is this text about port 3000 accurate?